sorc/build_fv3.sh

```diff
@@ -21,5 +21,5 @@ if [ $target = hera ]; then target=hera.intel ; fi
 cd fv3gfs.fd/
 FV3=$( pwd -P )/FV3
 cd tests/
-./compile.sh "$FV3" "$target" "NCEP64LEV=Y HYDRO=N 32BIT=Y" 1 YES YES
+./compile.sh "$FV3" "$target" "NCEP64LEV=Y HYDRO=N 32BIT=Y WW3=Y" 1 NO NO
```

Comment:
The following comment is from Jun and should be considered before merging into feature/gfsv16b.

Jun's comment follows:  

Well, I would suggest to use:

./compile.sh "$FV3" "$target" "32BIT=Y" 1

to build executable, by default, the last two arugment will always be YES, YES.  We do not use "NCEP64LEV" in the code and "HYDRO=N" is default compile option. 

To build with WW3:
  ./compile.sh "$FV3" "$target" "WW3=Y 32BIT=Y" 1

The last two arguments with value "NO" means not cleaning up before and after building the executable, we use it for debug when we do not want to rebuild everything from the beginning. I am not sure if the second executable is a clean build.

Reply:
We can have NCEP64LEV=Y removed. It was an option no longer used.  I'd prefer still keep HYDRO=N option so users know how to compile the model if they want to build with the hydrostatic option.

I had tried a few times to reset the last two options to YES in build_fv3.sh. Somehow it was always changed to NO at some point.

Reply:
Build ./compile.sh "$FV3" "$target" "WW3=Y 32BIT=Y" 1 has replaced previous form.